src/status_im/ui/screens/profile/contact/styles.cljs

```diff
@@ -0,0 +1,40 @@
+(ns status-im.ui.screens.profile.contact.styles
+  (:require-macros [status-im.utils.styles :refer [defstyle]])
+  (:require [status-im.ui.components.colors :as colors]))
+
+(def network-info {:background-color :white})
+
+(defstyle profile-setting-item
+  {:flex-direction :row
+   :align-items    :center
+   :padding-left   16
+   :ios            {:height 73}
+   :android        {:height 72}})
+
+(defn profile-info-text-container [options]
+  {:flex          1
+   :padding-right (if options 16 40)})
+
+(defstyle profile-info-title
+  {:color       colors/gray
+   :margin-left 16
+   :font-size   14
+   :ios         {:letter-spacing -0.2}})
```

Comment:
Can we abstract all those `letter-spacing` ?

Reply:
Maybe it can be done in a way similar to those uppercase texts. Not sure how exactly yet